identity-provider-service/src/bin/main.rs

```diff
@@ -400,7 +400,7 @@ impl DB {
             let file = std::fs::File::create(self.root.join("identity").join(key))?;
             let stored_obj = json!({
                 "identityObject": obj,
-                "accountAddress": AccountAddress::new(&obj.value.pre_identity_object.pub_info_for_ip.reg_id),
+                "accountAddress": account_address_from_registration_id(&obj.value.pre_identity_object.pub_info_for_ip.reg_id),
```

Comment:
Ah, I guess we don't have `Curve` defined in ccc.

Reply:
The `AccountAddress` type now lives in `concordium-contracts-common`, meaning we can't add methods directly, but only through traits.
Adding this as a method in `concordium-contracts-common` is not possible, since the function argument is a type from `concordium-base`.

Reply:
Once we define a proper type for `CredId` we can implement an `Into<AccountAddress>` for it.